### DISCUSSION TITLE
Requesting for Data Analytics panel

### DISCUSSION THREAD

**Discussion by saicsk-deloitte** [foxglove/community#593] [Q&A]
It could be better and very useful , if there would a panel in FOXGLOVE STUDIO TOOL ,which gives an overall information/analysis of given rosbag or db3 file (as shown in below - Sample format) 

**Note:** Attached image is just to show/say about the format which gives info about all the required things in graphical way (pie charts , bar graphs , data sets )

![Sample](https://user-images.githubusercontent.com/101085412/210172618-287506f9-2a7d-41cf-966e-421d42cf2056.png)

**Comment by amacneil** [comment]
What information about the bag/mcap file would you like to see?

**Reply by saicsk-deloitte** [reply]
Hi @amacneil , Firstly thanks for getting back .

I would like to see the same data like "_raw messages, state transitions , data source Info_" . But my suggestion is this data has to be in the form of pie charts, bar graphs etc.. ( which looks good to see) while playing the rosbag .
